util/timer/src/timer.rs

```diff
@@ -77,6 +89,20 @@ impl TimerLoop {
             }
         }
     }
+
+    pub fn stop(self) {
```

Comment:
Do we need it?

Reply:
Just for conclusiveness. Should we remove it? It doesn't affect other features unless calling it manually.

Reply:
Would you make it in other PR?
I don't want it blocks merging other PR.

Reply:
Okay. I seperated PR.